examples/get_bucket_encryption.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+# MinIO Python Library for Amazon S3 Compatible Cloud Storage.
+# Copyright (C) 2020 MinIO, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Note: YOUR-ACCESSKEYID, YOUR-SECRETACCESSKEY and my-bucketname are
+# dummy values, please replace them with original values.
+
+from minio import Minio
+from minio.error import ResponseError
+
+CLIENT = Minio('s3.amazonaws.com',
```

Comment:
All capital letters usually means a constant, I think you should replace 'CLIENT' with 'client'

Reply:
I have Python lint running and that was the recommendation. 
I thought we follow and adjust our code according to lint warnings. 
I'll change them back to lower case.
Could you also please confirm that we don't follow Python lint recommendations?
Thanks